tools/tune/crossover/crossover_build_blob.m

```diff
@@ -0,0 +1,79 @@
+function blob8 = crossover_build_blob(blob_struct, endian)
```

Comment:
this might need reworking to fit the new ABI if i remember correctly.

Reply:
Could you explain more? Thanks

Reply:
Given the crossover is built up of that struct that we want to remove that Seb created, the format of the crossover is likely to change so this function will likely need updated to match that new blob format. You will likely be able to call the iir blob build for the subsections. This will also make the crossover resiliant to any abi changes in the IIR.

Reply:
There is a chance I am totally wrong about this, I am just trying to future proof the ABI but this might be the smarter solution